Readme.md

```diff
@@ -37,33 +37,81 @@ Find, analyze, and remediate vulnerabilities present in your container images.
 
 | Regisry | Status |
 |--------------|:-----------------:|
-| Docker Hub | [In progress ⏳](https://github.com/shinobistack/gokakashi/issues/81) |
-| Google Artifact Registry | [In progress ⏳](https://github.com/shinobistack/gokakashi/issues/82) |
+| Docker Hub | ✅ [Enhancement in progress ⏳](https://github.com/shinobistack/gokakashi/issues/81) |
+| Google Artifact Registry | ✅ [Enhancement in progress ⏳](https://github.com/shinobistack/gokakashi/issues/82) |
```

Comment:
This renders like

![image](https://github.com/user-attachments/assets/3b64c8ad-23e4-4f64-af95-c0771f7366f9)

----

The tick mark usually co-relates to a "done" item in my mind and the hour-glass co-relates a "in-progress" item. So better to have only one of them.

so

"In progress :hourglass_flowing_sand:" 

vs

"Enhancement in progress :hourglass_flowing_sand:"

I would definitely vote for the shorter version: "In progress :hourglass_flowing_sand:"